### PR TITLE
first ocp_reveal release

### PR DESCRIPTION
**Pull request by zapashcanon**
Hi,

Here's the first release of [ocp_reveal](https://github.com/OCamlPro/ocp-reveal).

Cheers !

### CODE REVIEW

**Comment by zapashcanon** [comment]
The Travis failure on trunk seems to be unrelated.

File: packages/ocp_reveal/ocp_reveal.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+synopsis: "OCaml bindings for Reveal.js, an HTML presentation framework"
+description: "OCaml bindings for Reveal.js, an HTML presentation framework"
+maintainer: ["OCamlPro <contact@ocamlpro.com>"]
+authors: ["Çagdas Bozman"]
+license: "LGPL"
+homepage: "https://github.com/OCamlPro/ocp-reveal"
+doc: "github.com/OCamlPro/ocp-reveal"
+bug-reports: "https://github.com/OCamlPro/ocp-reveal/issues"
+depends: [
+  "ocaml" {>= "4.05"}
```
Comment (mseri) [inline packages/ocp_reveal/ocp_reveal.0.1/opam:11]:
```suggestion
  "ocaml" {>= "4.05" & < "4.11"}
```
The failure seems to indicate incompatibility with ocaml 4.11

Comment (mseri) [reply packages/ocp_reveal/ocp_reveal.0.1/opam:11]: Because in that case I would merge directly, without changes

Comment (zapashcanon) [reply packages/ocp_reveal/ocp_reveal.0.1/opam:11]: I don't see why it wouldn't work with 4.11.

**Comment by zapashcanon** [comment]
Thanks !

**Comment by avsm** [comment]
Just one minor point - did you intend to call the package `ocp_reveal` instead of `ocp-reveal`? The convention is to use dashes and not underscores in opam package names.

**Comment by zapashcanon** [comment]
It was intended. I think `-` are confusing because you can't use them everywhere. Whereas `_` can be used in opam packages, libraries and modules names. Ideally I'll try to rename the GitHub repository to ocp_indent but later.

```sh
cd opam-repository
ls | grep _ -c
332
ls | grep - -c
1167
```

The conventions is not completely clear. There's still hope for `_`. :D

**Pull request merged**